python pratices/assignment.12.py

```diff
@@ -0,0 +1,11 @@
+original_list=[0,10,[20,30],40,50,[60,70,80],[90,100,110,120]
+flatten_list = []
+ def removenesting(original_list):
+for i in original_list:

```

Comment:
No indentation after defining a function. Aren't you running these programs before sending for review?